pdr_backend/pdr_dashboard/test/test_util_db.py

```diff
@@ -1,25 +1,25 @@
 from unittest.mock import patch
 
 import pandas
+
 from enforce_typing import enforce_types
 
 from pdr_backend.lake.subscription import Subscription
 from pdr_backend.util.time_types import UnixTimeMs
 
 
 @enforce_types
-@patch("pdr_backend.pdr_dashboard.util.db.DuckDBDataStore")
-def test_query_db(
-    mock_duckdb_data_store,
-    _sample_app,
-):
+@patch("pdr_backend.pdr_dashboard.util.db.duckdb.execute")
+def test_query_db(mock_duckdb_execute, _sample_app):
     query = "query"
     db_mgr = _sample_app.data
     db_mgr._query_db(query)
 
-    lake_dir = db_mgr.lake_dir
-    mock_duckdb_data_store.assert_called_once_with(lake_dir, read_only=True)
-    mock_duckdb_data_store.return_value.query_data.assert_called_once_with(query)
+    # Assert that duckdb.execute was called once with the correct query
+    mock_duckdb_execute.assert_called_once_with(query)
+
+    # Mocking the return value from duckdb.execute(query)
+    mock_duckdb_execute.return_value.pl.assert_called_once()
```

Comment:
1. I'm not sure what this test is supposed to do, it doesn't seem to cover/assert/prove anything.
2. There is no test/coverage for parquet functionality.

Reply:
There were no test written yet until we decide what the final solution is